components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -559,3 +497,124 @@ func removeSensitiveCookies(cookies []*http.Cookie, domain string) []*http.Cooki
 	}
 	return cookies[:n]
 }
+
+// region blobserve transport
+type blobserveTransport struct {
+	http.RoundTripper
+	*ideRoutes
```

Comment:
do you really want to embed `ideRoutes` or just keep a reference?

Reply:
a reference should be fine